workers/worker_scheduler.rb

```diff
@@ -3,10 +3,13 @@
 module FastlaneCI
   # Class that handles the scheduling for fastlane.ci workers
   class WorkerScheduler
```

Comment:
Might just remove this class later and do this directly in the scheduler method.